CONTRIBUTING.md

```diff
@@ -53,18 +53,18 @@ Build and Test cycles are run on every commit to every branch using [GitHub Acti
 | Requirement | Tested Version | Installation Instructions                                                                      |
 | ----------- | -------------- | ---------------------------------------------------------------------------------------------- |
 | Node.js     | 18.16.0        | [Introduction to Node.js](https://nodejs.dev/en/learn/)                                        |
-| NPM         | 9.6.3          | [NPM Package Manager](https://nodejs.dev/en/learn/an-introduction-to-the-npm-package-manager/) |
+| PNPM         | 8.15.3          | [PNPM Package Manager](https://pnpm.io/installation) |
```

Comment:
```suggestion
| Node.js     | 18.16.0         | [Introduction to Node.js](https://nodejs.dev/en/learn/)                                        |
| PNPM        | 8.15.3          | [PNPM Package Manager](https://pnpm.io/installation)                                    |
```

Reply:
Addressed in [4442df4](https://github.com/TBD54566975/web5-js/pull/419/commits/4442df4f47b676650cbb74fa85b0c48917fc600f)